compiler-plugin/src/main/scala/scalapb/package.scala

```diff
@@ -1,11 +1,13 @@
+import com.trueaccord.scalapb.compiler.ProtobufGenerator
 import protocbridge.JvmGenerator
 
 package object scalapb {
   def gen(
     flatPackage: Boolean = false,
     javaConversions: Boolean = false,
     grpc: Boolean = true,
-    singleLineToString: Boolean = false): (JvmGenerator, Seq[String]) =
+    singleLineToString: Boolean = false,
+    collectionType: String = "scala.collection.Seq"): (JvmGenerator, Seq[String]) =
```

Comment:
A set of three options would be the best in my opinion.
In some cases I would really like to use plain arrays of primitives for the performance reasons, for example.

So, options having priorities from high to low in the following order would be really the perfect variant:
Field level option -> File-level option -> Global option -> Seq